docs/src/main/sphinx/release/release-393.md

```diff
@@ -0,0 +1,116 @@
+# Release 393 (10 August 2022)
+
+## General
+
+* Add support for the SQL [Merge](/sql/merge) statement. ({issue}`7933`)
+
+## Security
+
+* Fix potential incorrect rejection when using OAuth 2.0 refresh tokens. ({issue}`13168`)
```

Comment:
Include the information of when this can happen. Please check with @martint however.

```suggestion
* Fix potential incorrect rejection of tokens when OAuth 2.0 refresh-tokens are not obtained from IdP but were expected. ({issue}`13168`)
```